libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -79,6 +79,15 @@ export class PageContentDirective {
     return this.config && this.config.fixed;
   }
 }
+@Component({
+  selector: 'kirby-page-header',
```

Comment:
I could use some feedback on what to call this component, which purpose is to be embedded in the modal header start slot. However, I don't feel like kirby-page-header-start-slot is a very good name albeit descriptive.

Reply:
We've decided one kirby-page-progress to denote the component name with the most descriptive purpose for its intended use.